src/Dialog/Dialog.js

```diff
@@ -46,6 +47,111 @@ export const styleSheet = createStyleSheet('MuiDialog', (theme) => {
   };
 });
 
+type MaxWidthBreakpoints = 'xs' | 'sm' | 'md';
```

Comment:
To verify but I don't think that https://github.com/brigand/babel-plugin-flow-react-proptypes can handle the indirection resulting in `.any`.